src/Speckle.Objects/Data/NavisworksObject.cs

```diff
@@ -0,0 +1,20 @@
+using Speckle.Sdk.Models;
+
+namespace Speckle.Objects.Data;
+
+/// <summary>
+/// Represents a "first selectable ancestor" Navisworks.ModelItem object in Navisworks
+/// </summary>
+[SpeckleType("Objects.Data.NavisworksObject")]
+public class NavisworksObject : Base, INavisworksObject
+{
+  public required string name { get; set; }
+
+  public required List<Base> displayValue { get; set; }
```

Comment:
`[DetachedProperty]` should be on all `displayValue` properties (also missing from several other classes)